entry_check.php

```diff
@@ -0,0 +1,31 @@
+<?php
+require_once('functions.php');
+session_start();
+
+$db = getDB();
+$science_name = $_POST['science_name'];
+$name = $_POST['name'];
+$type = $_POST['type'];
+
+if (empty($science_name) || empty($name) || empty($type)) {
+    header('Location: addpage.php?entry_add_error=1');
+} else {
+    $nonunique_entry = checkforUniqueAddEntry($db, $science_name, $name);
+    $used_type = checkForWrongPlantType($db, $type);
+
+    if($nonunique_entry === false && $used_type !== ['not found']) {
```

Comment:
missing a space after the `if` before the `(`

Reply:
also if your cimparing a boolean to false. easier to just do: `!$nonunique_entry`